pymc/sampling/forward.py

```diff
@@ -459,21 +459,23 @@ def sample_posterior_predictive(
     Parameters
     ----------
     trace : backend, list, xarray.Dataset, arviz.InferenceData, or MultiTrace
-        Trace generated from MCMC sampling, or a list of dicts (eg. points or from find_MAP()),
-        or xarray.Dataset (eg. InferenceData.posterior or InferenceData.prior)
+        Trace generated from MCMC sampling, or a list of dicts (eg. points or from :func:`~pymc.find_MAP`),
+        or :class:`xarray.Dataset` (eg. InferenceData.posterior or InferenceData.prior)
     model : Model (optional if in ``with`` context)
         Model to be used to generate the posterior predictive samples. It will
         generally be the model used to generate the ``trace``, but it doesn't need to be.
-    var_names : Iterable[str]
+    var_names : Iterable[str], optional
         Names of variables for which to compute the posterior predictive samples.
+        By default, only observed variables are sampled.
+        See the example below for what happens when this argument is customized.
     sample_dims : list of str, optional
         Dimensions over which to loop and generate posterior predictive samples.
-        When `sample_dims` is ``None`` (default) both "chain" and "draw" are considered sample
-        dimensions. Only taken into account when `trace` is InferenceData or Dataset.
+        When ``sample_dims`` is ``None`` (default) both "chain" and "draw" are considered sample
+        dimensions. Only taken into account when ``trace`` is InferenceData or Dataset.
```

Comment:
references to arguments should have only one backtick, everything else we want formatted as code two according to numpydoc (and it is planned to use that to link/highlight between arguments in the future)